app/views/shared/_navbar.html.erb

```diff
@@ -29,7 +29,7 @@
         </li>
       <% else %>
         <li class="nav-item">
-          <%= link_to "Login", new_user_session_path, class: "nav-link" %>
+          <%= link_to "Log-in", new_user_session_path, class: "nav-link" %>
```

Comment:
Hey @nachoal the login text looks weird, what do you think about using the text 'sign-in'?